pages/unlock-publishing.tsx

```diff
@@ -0,0 +1,256 @@
+                        hover:text-white-default hover:bg-blue-light active:text-white-default active:bg-blue-active active:top-0.5
+                        focus:outline-[3px] focus:outline-blue-focus focus:outline focus:outline-offset-2 focus:bg-blue-focus focus:text-white-default disabled:cursor-not-allowed disabled:text-gray-500
+                      `}
+                      disabled={submitting}
+                    >
+                      {t("submitButton", { ns: "common" })}
+                    </Button>
+                    <StyledLink
+                      href={`/${i18n.language}/myforms/`}
+                      className={` 
+                          no-underline visited:text-black-default 
+                          bg-white-default text-black-default border-black-default py-4 px-8 rounded-lg border-2 border-solid
+                          hover:text-white-default hover:bg-blue-light active:text-white-default active:bg-blue-active active:top-0.5
+                          focus:outline-[3px] focus:outline-blue-focus focus:outline focus:outline-offset-2 focus:bg-blue-focus focus:text-white-default disabled:cursor-not-allowed disabled:text-gray-500
+                        `}
+                    >
+                      {t("unlockPublishing.skipStepButton")}
+                    </StyledLink>
+                  </div>
+                </form>
+              </>
+            )}
+          </Formik>
+        </>
+      )}
+      {hasRequestedPublishing && (
+        <>
+          <h1>{t("unlockPublishingSubmitted.title")}</h1>
+          <h2>{t("unlockPublishingSubmitted.whatNext.title")}</h2>
+          <p>{t("unlockPublishingSubmitted.whatNext.paragraph1")}</p>
+          <p className="mt-8 font-bold">{t("unlockPublishingSubmitted.whatNext.paragraph2")}</p>
+          <p>{t("unlockPublishingSubmitted.whatNext.paragraph3")}</p>
+          <p className="mt-8">{t("unlockPublishingSubmitted.whatNext.paragraph4")}</p>
+          <div className="flex mt-14">
+            <StyledLink
+              href={`/${i18n.language}/myforms/`}
+              className={` 
+                no-underline visited:text-white-default mr-8
+                bg-blue-dark text-white-default border-black-default py-4 px-8 rounded-lg border-2 border-solid
+                hover:text-white-default hover:bg-blue-light active:text-white-default active:bg-blue-active active:top-0.5
+                focus:outline-[3px] focus:outline-blue-focus focus:outline focus:outline-offset-2 focus:bg-blue-focus focus:text-white-default disabled:cursor-not-allowed disabled:text-gray-500
+              `}
+            >
+              {t("continue", { ns: "common" })}
+            </StyledLink>
+          </div>
+        </>
+      )}
+    </>
+  );
+}
+
+export const getServerSideProps = requireAuthentication(async ({ user: { ability }, locale }) => {
+  {
+    checkPrivileges(ability, [{ action: "view", subject: "FormRecord" }]);
```

Comment:
This may not be the right privilege/way to check if a new user can request publishing access.

Reply:
To check if a user has publishing ability you can use `ability.can("update", "FormRecord", "isPublished");`